Windows tests failing in CI

Error:

```
Run git clone https://***@github.com/fractl-io/fractl-config-secrets-reader
Cloning into 'fractl-config-secrets-reader'...
Error: Could not find or load main class "-Dclojure.compile.path=D:\a\fractl\fractl\fractl-config-secrets-reader.target.classes"
Caused by: java.lang.ClassNotFoundException: "-Dclojure.compile.path=D:\a\fractl\fractl\fractl-config-secrets-reader.target.classes"
Error: Process completed with exit code 1.
```

--- a/.github/workflows/fractl-clj.yml
+++ b/.github/workflows/fractl-clj.yml
@@ -14,7 +14,7 @@ jobs:
   test_clj:
     strategy:
       matrix:
-        platform: [ windows-latest ]
+        platform: [ ubuntu-latest, windows-latest ]
     runs-on: ${{ matrix.platform }}
     defaults:
       run:

--- a/.github/workflows/fractl-clj.yml
+++ b/.github/workflows/fractl-clj.yml
@@ -14,27 +14,29 @@ jobs:
   test_clj:
     strategy:
       matrix:
-        platform: [ ubuntu-latest, windows-latest ]
+        platform: [ windows-latest ]
     runs-on: ${{ matrix.platform }}
 
     steps:
       - name: Setup postgres
         uses: ikalnytskyi/action-setup-postgres@v4
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Prepare java
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
-          distribution: "zulu"
-          java-version: "19"
+          distribution: 'zulu'
+          java-version: '17'
       - name: Install clojure tools
-        uses: DeLaGuardo/setup-clojure@12.1
+        uses: DeLaGuardo/setup-clojure@12.3
         with:
           lein: 2.9.1 # Leiningen
-      - name: Installing fractl-io dependancies
+      - name: Installing fractl-io dependencies
         run: |
+          lein -v
           git clone https://${{ secrets.REPO_READ_WRITE_TOKEN }}@github.com/fractl-io/fractl-config-secrets-reader
           cd fractl-config-secrets-reader
+          ls
           lein install
       - name: Install dependencies
         run: lein deps

--- a/.github/workflows/fractl-clj.yml
+++ b/.github/workflows/fractl-clj.yml
@@ -18,7 +18,7 @@ jobs:
     runs-on: ${{ matrix.platform }}
     defaults:
       run:
-        shell: cmd
+        shell: ${{ matrix.platform == 'windows-latest' && 'cmd' || 'bash' }}
 
     steps:
       - name: Setup postgres

--- a/.github/workflows/fractl-clj.yml
+++ b/.github/workflows/fractl-clj.yml
@@ -14,7 +14,7 @@ jobs:
   test_clj:
     strategy:
       matrix:
-        platform: [ubuntu-latest]
+        platform: [ ubuntu-latest, windows-latest ]
     runs-on: ${{ matrix.platform }}
 
     steps:

--- a/.github/workflows/fractl-clj.yml
+++ b/.github/workflows/fractl-clj.yml
@@ -16,6 +16,9 @@ jobs:
       matrix:
         platform: [ windows-latest ]
     runs-on: ${{ matrix.platform }}
+    defaults:
+      run:
+        shell: cmd
 
     steps:
       - name: Setup postgres
@@ -26,7 +29,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
-          java-version: '17'
+          java-version: '19'
       - name: Install clojure tools
         uses: DeLaGuardo/setup-clojure@12.3
         with:
@@ -36,7 +39,6 @@ jobs:
           lein -v
           git clone https://${{ secrets.REPO_READ_WRITE_TOKEN }}@github.com/fractl-io/fractl-config-secrets-reader
           cd fractl-config-secrets-reader
-          ls
           lein install
       - name: Install dependencies
         run: lein deps

